### PR TITLE
More typos

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -46733,6 +46801,8 @@ quantititive->quantitative
 quantitity->quantity
 quantitiy->quantity
 quarantaine->quarantine
+quarantee->guarantee
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:46804]: very close to 'quarantine', could be a mistake for a non-english speaker perhaps ?

File: codespell_lib/data/dictionary_rare.txt [hunk 2]
```diff
@@ -232,6 +233,10 @@ readding->re-adding, reading,
 readds->re-adds, reads,
 ream->stream
 recuse->recurse
+reduct->deduct, reduce,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary_rare.txt:236]: also 'redact'

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:46804]: I haven't found such a typo out there:
https://grep.app/search?q=quarantee

File: codespell_lib/data/dictionary_rare.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary_rare.txt]: Added.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: just out of curiosity i through it into google:

Screenshot_20240909_071018

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: Such dictionary suggestions are based on a distance metric between words, but that distance metric doesn't necessarily reflect the probability of typos in practice. Anyway, I have added `quarantine`.

**Review by luzpaz — APPROVED** [review]

**Pull request merged**